addons/website_sale/models/product.py

```diff
@@ -430,6 +430,14 @@ class Product(models.Model):
 
     product_variant_image_ids = fields.One2many('product.image', 'product_variant_id', string="Extra Variant Images")
 
+    website_url = fields.Char('Website URL', compute='_compute_product_website_url', help='The full URL to access the document through the website.')
+
+    @api.multi
+    def _compute_product_website_url(self):
+        for product in self:
+            variants = ','.join(str(x) for x in product.attribute_value_ids.ids)
+            product.website_url = "%s#variants=%s" % (product.product_tmpl_id.website_url, variants)
```

Comment:
The word variants for the attributes is in fact confusing. We use "Products" for `product.template` records and "Variants" for `product.product` records. So using `variants` for variants attributes seems confusing to me. What do you think @JKE-be ?

Reply:
why not simply attr

Reply:
I suggested attributes in the first place.